src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+import java.util.Random;
+
 public class Duke {
+
+    public static void printSeparator() {
```

Comment:
I like the use of an extracted method to print the separator!

Reply:
Thank you!